lib/rules/no-invalid-attribute-name.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @author Doug Wade <douglas.b.wade@gmail.com>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'require the attributes to match the imported component name',
```

Comment:
```suggestion
        'report invalid attribute names',
```

Please run `npm run update` to update the docs afterwards.

Reply:
I tried to use this error message, but it doesn't seem to lint:

```
/path/to/eslint-plugin-vue/lib/rules/no-invalid-attribute-name.js
  14:20  error  `meta.docs.description` must match the regexp /^(enforce|require|disallow)/  eslint-plugin/require-meta-docs-description
```

I substituted a different one that does, but I am certainly open to changing it.